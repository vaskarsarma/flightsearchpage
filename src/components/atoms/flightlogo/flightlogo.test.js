import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import FlightLogo from "./flightlogo";

describe("<FlightLogo />", () => {
  let container;

  beforeEach(() => (container = shallow(<FlightLogo />)));

  it("<FlightLogo /> Rendered Correctly", () => {
    expect(shallowToJson(container)).toMatchSnapshot();
  });

  it("should render a <img />", () => {
    expect(container.find("img").length).toEqual(1);
  });
});
