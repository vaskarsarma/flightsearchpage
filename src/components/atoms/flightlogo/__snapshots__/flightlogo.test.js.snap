// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlightLogo /> <FlightLogo /> Rendered Correctly 1`] = `
<img
  aria-hidden="true"
/>
`;
